### DISCUSSION TITLE
Allow Supported Color Models to be Rendered in Markdown Files

### DISCUSSION THREAD

**Discussion by aisgbnok** [community/community#19840] [Other Feature Feedback, Questions, & Ideas]
### Description

Specific color formats will show a small visualization of the color: `#0969da`, `#ffffff`. I like this feature; however, it does not render in markdown files like a readme. Is there any specific reason for this limitation? I would appreciate this feature being extended to markdown files, as long as there isn't a particular reason for its omission. Thanks! 👍

**Comment by Alice52** [comment]
so, how to use it?  

it's just `#0969da` dot?

**Reply by aisgbnok** [reply]
Yes. It it's just `` `#0969da` `` and it provides a small visual rendering of the color. `#0969da`

However, this doesn't work in markdown files such as a README.

See, the [GitHub "Supported color models" docs.](https://docs.github.com/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#supported-color-models)

**Reply by Alice52** [reply]
Thanks for reply, may I get some tricks for colored font, such as red for highlight text?

due to github's markdown even cannot render html syntax for color font.

**Reply by aisgbnok** [reply]
Unfortunately not that I know of. As you stated GitHub's markdown customization is very limited.
